src/main/java/io/appform/dropwizard/actors/connectivity/RMQConnection.java

```diff
@@ -249,4 +254,8 @@ private Map<String, Object> getPriorityOpts(final ActorConfig actorConfig) {
         }
     }
 
+    private void setupObservers(final RMQConfig config,
```

Comment:
The actual publish method should also be called through an observer. There can be a chain of observers, that sequence needs to be created in this method